laboratory/inspector/src/main/kotlin/io/mehow/laboratory/inspector/Views.kt

```diff
@@ -71,3 +74,19 @@ internal var View.isGone: Boolean
   set(value) {
     visibility = if (value) GONE else VISIBLE
   }
+
+internal fun View.fixSystemBarInsets() {
```

Comment:
it is not the prettiest solution as it leaves white bar when notch is present, but it works 🤷